### ISSUE TITLE
Thanks! This is awesome.

### ISSUE THREAD

**Issue by johndierks** [bertique/SonosNPROneServer#1]
This isn't an issue, per se, but just wanted to say thanks for writing this. It works great, your docs made it a breeze to setup, and I appreciate you open sourcing the code.

Thanks a ton! 🍻

**Comment by bertique** [comment]
Thanks, so much! That's great to hear.Glad you are enjoying it.

Cheers

**Comment by FredrikNoren** [comment]
+1! This is great!

**Comment by kPOWz** [comment]
this is amazing !  whole family can listen to 'David Sedaris Reads _Santaland Diaries_'  on the Sonos now!

**Comment by strouja** [comment]
@johndierks I agree and just found this it works flawlessly on my system.   I have upgraded to Sonos S2 and gave my older S1 products to friends and family.  So all good on S2.  I set up by your instructions which were great and used the ip address of one of my speakers and the `NPROne` showed up fine on Mac, PC and iPhone and works across all speakers as just a regular sonos music service across all speakers.

So great job and I love it.

Don't forget to stay current with your java versions :)
